src/components/Footer/index.js

```diff
@@ -1,32 +1,76 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import styled from 'styled-components'
 import { Padding } from 'styled-components-spacing'
+import { Location } from '@reach/router'
 
 import { Grid, Row, Col } from '../../components/grid'
 import { SectionTitle } from '../../components/Typography'
+import GetInTouch from '../Common/GetInTouch'
 import OfficeListing from './OfficeListing'
 import FooterLinks from './FooterLinks'
 
 export const GreyFooter = styled.footer`
   background: #232323;
 `
 
+const Wrapper = styled.div`
+  background-color: ${({ theme, bgColor = 'greyBg' }) =>
+    theme.colors[bgColor || 'white']};
+`
```

Comment:
Since bgColor has a default value,  I believe that the 'white' should never be reached, right? Unless bgColor has a value that we don't have in theme.color; is that the reason we have the fallback to white?
PS: the destructuring of props is nice! I realise that we hardly do that in our styling. We should whenever possible.

Reply:
You're absolutely right thanks for spotting that

Reply:
you can remove the `|| 'white'`. Only `=> theme.colors[bgColor]` needed now 😉

Reply:
Oh sorry...that's what you did. Perfect. I was seeing an older version 😄